scss/login.scss

```diff
@@ -0,0 +1,66 @@
+@import './variables/_colors', './variables/_variables';
+
+.login {
+  align-items: center;
+  background-color: $white;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+}
+
+.login-container {
+  align-items: center;
+  background-color: $white-smoke;
+  border-top: 8px solid $sky-blue;
+  display: flex;
+  flex-direction: column;
+  height: $login-container-height;
+  width: $login-container-width;
+
+  &::before {
+    background-color: $dark-grey;
+    content: '';
+    height: 2px;
+    position: relative;
+    top: 333px;
+    width: 252px;
+  }
+}
+
+.login-label {
+  font-size: 16px;
+}
+
+.login-input {
+  height: $login-items-height;
+  width: $login-items-width;
+}
+
+.login-submit-button {
+  background-color: $light-green;
+  border-radius: 5px;
+  color: $white;
+  height: $login-items-height;
+  width: $login-items-width;
+}
+
+.login-signup-button {
+  background-color: $white-smoke;
+  border-radius: 5px;
+  color: $light-green;
+  height: $login-items-height;
+  width: $login-items-width;
```

Comment:
if you want, the buttons are sharing some properties, so you can use the feature `Extend/Inheritance` that provides sass